### ISSUE TITLE
Tutorials don't start at all

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#2708]
### Short Summary

The getting started wizard isn't starting after startup wizard finishes.  yes it's in my toolbox.  

NOTE: ALL tutorials aren't starting any more.

### Steps to Reproduce

- Start Komodo  with a fresh profile 
- proceed through Startup wizard
- make sure on the last page that you have "start the "getting started" Tutorial when..." is selected
- Finish

Alt.

- Just try starting a tutorial from the Toolbox, even a newly created one.

### Expected results

Getting started tutorial to start

### Actual results

Doesn't start

### Errors
So there is an error when you open the tutorial but the tutorial seems to open fine despite the error.  I'm guessing the logic doesn't work but it doesn't prevent the tutorial from opening and running.
```
[2017-07-25 11:20:15,142] [ERROR] tutorials: failed to load logic
-- EXCEPTION START --
SyntaxError: illegal character
+ stack
    @file:///C:/Users/cgcho/AppData/Local/Temp/ko-tutorial1.js:1:undefined
    this.loadLogic@chrome://tutorials/content/sdk/tutorials.js:221:24
    this.start@chrome://tutorials/content/sdk/tutorials.js:173:17
    this.onInvoke@chrome://tutorials/content/sdk/tutorials.js:44:9
    this.invoke_startTutorial@chrome://komodo/content/toolbox/toolbox2_command.js:116:5
    this.invokeTool@chrome://komodo/content/toolbox/toolbox2_command.js:1334:5
    this.onDblClick@chrome://komodo/content/toolbox/toolbox2_command.js:1380:5
    ondblclick@chrome://komodo/content/toolbox/toolboxPanel.xul:1:1
-- EXCEPTION END --
```

There is also this error now:

```
2017-08-02 11:16:18,884] [ERROR] console-logger: TypeError: text is undefined (2) in chrome://tutorials/content/sdk/tutorials.js:709
Traceback (most recent call last):
  File "chrome://tutorials/content/sdk/tutorials.js", line 709, in 
```

**Comment by cgchoffman** [comment]
The logic error will be fixed when #2742 is fixed.

**Comment by cgchoffman** [comment]
https://github.com/Komodo/KomodoIDE/commit/1ab70f7fa030d951f997d06afffb2fe37ce28f19
